web/sites/default/config/views.view.admin_groups.yml

```diff
@@ -101,22 +101,28 @@ display:
           summary: ''
           description: ''
           columns:
+            views_bulk_operations_bulk_form: views_bulk_operations_bulk_form
             id: id
             label: label
             type: type
-            field_phone: field_phone
-            field_email: field_email
-            field_facebook: field_facebook
-            field_instagram: field_instagram
-            field_twitter: field_twitter
-            field_linkedin_link: field_linkedin_link
-            field_nextdoor_link: field_nextdoor_link
-            field_youtube_link: field_youtube_link
-            field_contact: field_contact
-            edit_group: edit_group
+            field_migration_status: field_migration_status
```

Comment:
This list of columns changed dramatically with totally different kinds of data. Is that expected?